lib/app.ts

```diff
@@ -77,11 +77,13 @@ export function setMaxListeners(maxListeners: number) {
 /** Selected CLI initialization steps */
 async function init() {
 	if (process.env.BALENARC_NO_SENTRY) {
-		console.error(`WARN: disabling Sentry.io error reporting`);
+		if (process.env.DEBUG) {
+			console.error(`WARN: disabling Sentry.io error reporting`);
+		}
 	} else {
 		await setupSentry();
 	}
-	checkNodeVersion();
+	await checkNodeVersion();
 
 	const settings = new CliSettings();
```

Comment:
A few lines below this line in the `init()` function, there is a call to `notify()`:
```ts
	// check for CLI updates once a day
	(await import('./utils/update')).notify();
```
We should not make that call if `BALENARC_OFFLINE_MODE` is set.